presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -939,6 +947,16 @@ std::vector<std::string> PrestoServer::registerConnectors(
   return catalogNames;
 }
 
+void PrestoServer::registerSystemConnector() {
+  PRESTO_STARTUP_LOG(INFO) << "Registering system catalog "
+                           << " using connector SystemConnector";
+  VELOX_CHECK(taskManager_);
+  auto systemConnector =
+      std::make_shared<SystemConnector>("$system@system", taskManager_.get());
```

Comment:
Should we use `$system` to match the `presto_protocol.yml` change?

Reply:
I actually changed presto_protocol.yml to $system@system as that seemed to work out better.